### PR TITLE
[Merged by Bors] - fix problem with not setup legacy tenant

### PR DESCRIPTION
**Pull request by rustonaut**
We should never hit that in practice with how we currently do things.

But better safe then sorry.

### CODE REVIEW

File: web-api-db-ctrl/src/elastic.rs [hunk 1]
```diff
@@ -48,21 +48,35 @@ pub(crate) async fn setup_legacy_tenant(
...
-              "add": {
-                "index": default_index,
-                "alias": tenant_id,
-              }
-            }
-          ]
-        }))
+    let response = elastic
+        .with_index(default_index)
+        .request(Method::GET, [], [])
```
Comment (janpetschexain) [inline web-api-db-ctrl/src/elastic.rs:53]: ❓ should we use a `Method::HEAD` request instead to have an even more light-weight "ping" response?
```suggestion
        .request(Method::HEAD, [], [])
```

**Review by janpetschexain — APPROVED** [review]
lgtm

File: web-api-db-ctrl/src/elastic.rs (hunk 1 above)
Comment (rustonaut) [reply web-api-db-ctrl/src/elastic.rs:53]: during some manual testing this didn't work correctly once for me as elastic did hung on the response for whatever reason :see_no_evil: 

also this is done once during startup, the size difference of the response really doesn't matter

**Comment by rustonaut** [comment]
bors merge

**Pull request closed without merging**